clearml_agent/helper/repo.py

```diff
@@ -225,6 +225,7 @@ def remote_branch_name(branch):
         (?:(?P<user>{regular}*?)@)?
         (?P<host>{regular}*?)
         :
+        (?:v3/)? # present in azure ssh urls
```

Comment:
@mads-oestergaard how is this actually used?

Reply:
It keeps :v3/ out of the `path` group.

Reply:
It's used down in line 254